packages/nx/src/command-line/migrate.ts

```diff
@@ -947,6 +947,7 @@ export async function executeMigrations(
         logger.info(`  ${m.description}\n`);
         printChanges(changes, '  ');
       } else {
+        require('../adapter/compat');
```

Comment:
This is done every single migration that is not `nx`. Can we have it happen once if `some` migrations are not `nx`?